textile/chat-features.textile

```diff
@@ -237,14 +260,18 @@ Each chat room can be configured individually, allowing options to be passed as
 *** @(CHA-RC1g5)@ @[Testable]@ The room is removed from the room map and a @CHA-RL3@ release operation is initiated for the room object. A future is returned which resolves when the release operation completes.
 * @(CHA-RC2)@ Chat rooms are configurable, so as to enable or disable certain features. When requesting a room, options as to which features should be enabled, and the configuration they should take, must be provided (@RoomOptions@).
 ** @(CHA-RC2a)@ @[Testable]@ If a room is requested with invalid configuration, for example: a negative typing timeout, an @ErrorInfo@ with code @40001@ must be thrown.
-** @(CHA-RC2c)@ @[Testable]@ A room’s messages feature (@CHA-M*@) can not be disabled.
-** @(CHA-RC2b)@ @[Testable]@ If a room is configured to have a given feature disabled, then attempting to use that feature must result in an @ErrorInfo@ with code @40000@ being thrown.
-** @(CHA-RC2d)@ @[Testable]@ A room must create a lifecycle contributor for each of its enabled features and must not create a lifecycle contributor for any of its disabled features.
-** @(CHA-RC2e)@ @[Testable]@ The room lifecycle contributors have an order of precedence, which is the order in which operation sequences (e.g. attaches are applied to them). The order of precedence shall be: messages, presence, typing, reactions, occupancy.
-** @(CHA-RC2f)@ @[Testable]@ A room must only fetch a realtime channel if this channel is required by one of its lifecycle contributors.
-* @(CHA-RC3)@ Multiple features may share one realtime channel (for example, presence and messages). Each feature may have different channel modes/options that need to be applied.
-** @(CHA-RC3a)@ @[Testable]@ When retrieving a channel via @channels.get()@, features that share a channel must merge together their respective options/modes and pass the merged result to the call to @channels.get()@.
+** @(CHA-RC2c)@ This specification point has been removed. It was valid up until the single-channel migration.
+** @(CHA-RC2b)@ This specification point has been removed. It was valid up until the single-channel migration.
+** @(CHA-RC2d)@ This specification point has been removed. It was valid up until the single-channel migration.
+** @(CHA-RC2e)@ This specification point has been removed. It was valid up until the single-channel migration.
+** @(CHA-RC2f)@ This specification point has been removed. It was valid up until the single-channel migration.
+* @(CHA-RC3)@ Multiple features share one realtime channel (for example, presence and messages). Each feature may have different channel modes/options that need to be applied.
+** @(CHA-RC3a)@ @[Testable]@ When retrieving a channel via @channels.get()@, features must merge together their respective options/modes and pass the merged result to the call to @channels.get()@.
```

Comment:
I see you've changed "features that share a channel" to "features" here. This implies that _all_ features share a channel. If this is the case, then I think that we should:

1. make this explicit in a single spec point that states the channel name
2. remove the per-feature spec points (e.g. CHA-M1) that state which channel to use

Reply:
Yes, I have done this - added CHA-RC3c and removed feature-specific points in 151c0904